src/snowflake/snowpark/modin/plugin/_internal/resample_utils.py

```diff
@@ -359,21 +359,25 @@ def perform_resample_binning_on_frame(
         The internal frame with a single DatetimeIndex column
         to perform resample binning on.
 
+    datetime_index_col : str
+        The datetime-like column to use for resampling.
```

Comment:
nit: can you please also mention if this is identifier or label for improved readability.

Reply:
Good point, done